README.md

```
@@ -1,20 +1,264 @@
+[![Code Smells](https://sonarcloud.io/api/project_badges/measure?project=antoinezanardi_goat-it-api&metric=code_smells)](https://sonarcloud.io/summary/new_code?id=antoinezanardi_goat-it-api)
+
+## 📖 Table of Contents
+
+<!-- TOC -->
+
+- [🐐 What is this API ?](#what-is-this-api)
+- [🔨 Installation](#installation)
+- [🚀 Build](#build)
+- [🐳 Docker](#docker)
+- [💯 Tests](#tests)
+  - [🧪 Unit tests](#unit-tests)
```

markdownlint-cli2 (0.18.1)

Link fragments should be valid

(MD051, link-fragments)